docs/1.2-DRAFT/root-data-entity.md

```diff
@@ -44,7 +44,7 @@ The _RO-Crate JSON-LD_ MUST contain a self-describing
 **RO-Crate Metadata File Descriptor** with
 the `@id` value `ro-crate-metadata.json` (or `ro-crate-metadata.jsonld` in legacy
 crates) and `@type` [CreativeWork]. This descriptor MUST have an [about]
```

Comment:
We need to clarify that the descriptor's id can also be an absolute URI. For instance, by add a sentence here like:

```
In a [detached RO-Crate](structure.md#detached-ro-crate), the descriptor's `@id` can be an absolute URI; in this case, its last path segment MUST be `ro-crate-metadata.json` (or `ro-crate-metadata.jsonld` in legacy crates)
```

Reply:
An alternative is rephrasing the previous sentence as:

```
The _RO-Crate JSON-LD_ MUST contain a self-describing **RO-Crate Metadata File Descriptor**
whose `@id` MUST have `ro-crate-metadata.json` (or `ro-crate-metadata.jsonld` in legacy crates)
as its last path segment.
```